dist/static/site.js

```diff
@@ -7,104 +7,52 @@ let sock = null;
 /** Heartbeat variable */
 let isAlive = true;
 
-function removeChildren(parent) {
-  while (parent.firstChild) {
-    parent.removeChild(parent.firstChild);
-  }
-}
-
-function createSong(song) {
-  const div = document.createElement('div');
-  div.className = 'song';
-
-  const divImage = document.createElement('div');
-  divImage.className = 'song-thumb';
-
-  const aThumb = document.createElement('a');
-  aThumb.href = song.webpage_url;
-  aThumb.target = '_blank';
-
-  const imgThumb = document.createElement('img');
-  imgThumb.src = song.thumbnail;
-
-  divImage.appendChild(imgThumb);
-
-  aThumb.appendChild(divImage);
-  div.appendChild(aThumb);
-
-  const songTitleDiv = document.createElement('div');
-  songTitleDiv.className = 'songtitle';
-  songTitleDiv.appendChild(document.createTextNode(song.title));
-
-  div.appendChild(songTitleDiv);
-
-  return div;
+function createSong(song, onclick, username = null) {
+  return `
+    <div class="song">
+      <a target="_blank" href=${song.webpage_url}>
+        <div class="song-thumb">
+          <img src=${song.thumbnail}>
+        </div>
+      </a>
+      <div class="songtitle">
+        ${song.title}
+      </div>
+      ${username === null || username === name ? `<button class="b-remove" onclick='${onclick}'></button>` : ''}
```

Comment:
${username === null || username === name ? `<button class="b-remove" onclick="${onclick}"></button>` : ''}

Reply:
What was the change requested here, I can't see it

Reply:
Changing the '' to ""

Reply:
Ah, I'm going to keep this this way though, since this is the eslint default.

Reply:
Oh sorry, I meant the single quotes on the onclick attribute